mpact_api/mpact/services.py

```diff
@@ -1,49 +1,71 @@
-from telegram.ext import Updater
+from telethon import TelegramClient
 
-from .constants import BOT_TOKEN, ID, MESSAGE, REPLACE, WEBHOOK_URL
-from .models import UserChat, UserData
-from .serializers import ChatDataSerializer, UserDataSerializer
+from .constants import (
+    BOT_TOKEN,
+    CODE,
+    LOGOUT,
+    MESSAGE,
+    MESSAGE_SENT,
+    NOT_AUTHORIZED,
+    PHONE,
+    TELEGRAM_API_HASH,
+    TELEGRAM_API_ID,
+)
+from .logger import logger
 
-updater = Updater(BOT_TOKEN, use_context=True)
-dispatcher = updater.dispatcher
+# Remember to use your own values from my.telegram.org!
+api_id = TELEGRAM_API_ID
+api_hash = TELEGRAM_API_HASH
+bot_token = BOT_TOKEN
+# loop = asyncio.get_event_loop()
+client = TelegramClient("anon", api_id, api_hash)
+bot = TelegramClient("bot", api_id, api_hash).start(bot_token=bot_token)
```

Comment:
Are you sure you want to instantiate your clients when the module is loaded? Telegram will remember your session between requests. You don't need to instantiate these at load time.

```suggestion
def get_anon_client() -> TelegramClient:
    """
    Returns a TelegramClient with the session name "anon"

    .. IMPORTANT::
       Exposing a logged-in session over a web API allows **anyone** to
       use that session, not just the person who logged in!

    """
    return get_telegram_client("anon")


def start_bot_client() -> TelegramClient:
    return get_telegram_client("bot").start(bot_token=BOT_TOKEN)


def get_telegram_client(session_name: str) -> TelegramClient:
    return TelegramClient(session_name, TELEGRAM_API_ID, TELEGRAM_API_HASH)
```